src/armature/Node.ts

```diff
@@ -62,26 +67,181 @@ export class Node {
         return point;
     }
 
+    /**
+     * Holds the current node in place at a given point so that it can be manipulated about
+     * that point.
+     *
+     * @param {Point | vec3} point The point to be held, either as a local coordinate, or a
+     * control point on the current or any other node.
+     *
```

Comment:
This is a nitpick, but could we please remove the space in between the `@param`, and `@returns`, and just have the one space after the initial description?